### PR TITLE
Docs ready for rtd, issue templates, docstrings and build/publish workflow

### PR DESCRIPTION
**Pull request by leonhma**
Hi there,
Marcel told me about this repo and i figured i would contribute to the automation/docs. So in this PR I did several things:
* Documentation using sphinx, ready to deploy with readthedocs. You can see it, including docs preview for pull requests on my forked repo.
* Docstrings and typehints in the code. Works with IDEs, too!
* Workflow to publish the package to (test-)pypi if a new release is created.
    To use this feature, go to pypi and testpypi and create keys, that have to be stored in the repos secrets as 'TEST_PYPI_PASSWD' and 'PYPI_PASSWD'. If you enable 'This is a prerelease' while creating a release, the built package will only be deployed to testpypi. The tag name (version) has to follow pep440 and should use semver.
* Issue templates. Nicer looking forms to fill out, guiding the process of filing an issue.
* For added karma, this:
    [Documentation Status](https://lemon-markets-api-access-poc.readthedocs.io/en/latest/?badge=latest)

Also, i made the code comply with several pep standards. Guys i urge you to use a linter (pylama for example).

### CODE REVIEW

File: lemon_markets/market_data.py [hunk 1]
```diff
@@ -1,26 +1,67 @@
...
+        -------
+        Union[dict, pandas.DataFrame]
+            Either the raw response json data (as dict) or a pandas dataframe
+
+        """
         endpoint = f"trading-venues/{venue.mic}/instruments/{instrument.isin}/data/ohlc/{x1}/"
         params = {}
         if ordering is not None:
-            params['ordering'] = ordering
+            # TODO what is the value the server accepts as True?
```
Comment (leonhma) [inline lemon_markets/market_data.py:63]: What *is* the value the server acceptsas True?

Comment (leonhma) [reply lemon_markets/market_data.py:63]: The relevant line of code is below this comment...

**Review by LinusReuter — APPROVED** [review]

**Comment by LinusReuter** [comment]
Thank you for all your work. I have looked throw the changes and will merge the pull request. 
For me it's my first public coding project, so I will take some learnings from your changes and comments, thanks. 

Linus

**Comment by LinusReuter** [comment]
I added the secrets like mentions and made a release, but there is an error ( twine upload: error: argument -p/--password: expected one argument ) while execution of the workflow. 
are you able to help me out with the setup?

**Comment by leonhma** [comment]
No worries, i'm quite new, too. As i said in the first 'post', the tag has to follow pep440. You can look up the scheme here: [pep440](https://www.python.org/dev/peps/pep-0440/#public-version-identifiers). So you can try `0.1rc1`. Tip: I have worked with the exact package name `lemon_markets` on pypi before, so some specific versions wont upload anymore bc pypi doesnt allow to overwrite files (even the ones that are deleted). I think `0.0.1`, `0.0.2`, `0.0.3` and `1.0.0` dont work 😬

**Comment by leonhma** [comment]
In addition, while updating my fork with the upstream changes, i noticed that `.pyproject.toml` breaks the build on rtd.

**Comment by LinusReuter** [comment]
Can you contact me on Slack (Linus R)? There we can have a deeper talk on some questions and I would make you a maintainer. 
As pypi package name I used lemon_markets_sdk. The build gets completed but I get an error in the upload process. 😬

**Pull request merged**